src/lasp_core.erl

```diff
@@ -77,6 +77,12 @@
                 ?MODULE:bind(_AccId, AccValue, _Store)
               end).
 
+-define(WRITE, fun(_Store) ->
```

Comment:
Write is only there to generate the bind function with the `Store` captured in a closure, so we don't have to pass that to the lasp process as part of the writing function